Makefile

```diff
@@ -53,3 +56,24 @@ release: clean publish build-proxy
 
 test:
 	echo "TODO: add tests"
+
+setup-environment:
+	@if [ -e /usr/bin/yum ]; then \
+		scripts/rhel_setup_environment_1.sh; \
```

Comment:
Now that you no longer need to log out and log back in against manually could these two scripts be collapsed?